cmd/device-util/service.go

```diff
@@ -0,0 +1,153 @@
+//go:build windows
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"sync"
+	"unsafe"
+
+	cli "github.com/urfave/cli/v2"
+	"golang.org/x/sys/windows"
+)
+
+var _scm scm
```

Comment:
Does this need to be global? I think we can also use https://pkg.go.dev/golang.org/x/sys/windows/svc/mgr#Mgr instead to manage the SCM connection.

Reply:
I didnt see that, Ill try that out